### ISSUE TITLE
broken paperswithcode url links in tensor.py

### ISSUE THREAD

**Issue by geohotstan** [tinygrad/tinygrad#11486]
https://x.com/julien_c/status/1948718297194561956
So now all paperswithcode links in tensor method documentation redirects to their github

Try it here: https://paperswithcode.com/method/mish

**Comment by chenyuxyz** [comment]
looks like it's shut down https://github.com/paperswithcode/paperswithcode-data/issues/45, so i am removing the links now

**Comment by b0r1ngx** [comment]
@geohotstan, @chenyuxyz Hello, maybe its more appropriate to save links with using this url's from web.archive.org.

For example for Batch Normalization we use: https://web.archive.org/web/20250409000547/https://paperswithcode.com/method/batch-normalization

Because description itself is necessary sometimes.
